README.md

```diff
@@ -1742,43 +1830,6 @@ Default: `0`
 let g:ycm_min_num_identifier_candidate_chars = 0
 ```
 
-### The `g:ycm_max_num_candidates` option
```

Comment:
i think either my branch, or your branch is out of date ? there are unexpected diffs ?

Reply:
Somehow, my branch was seriously out of date. That's fixed now.